### PR TITLE
Cheatsheets: Change format of keybindings having alternative keys.

### LINKED ISSUE #1407: Cheat Sheets: Add spacing around Parentheses and Slashes in Alternatives
We decide to add some padding around the alternative keys. We should updated all Goodies so that alternatives like `([A]/[B]/[C])` are spaced out to look like this: `( [A] / [B] / [C] )`.

This can done manually, but a small find+sed script should be able to handle it as well :smile: 

Any takers?

### PR DESCRIPTION
**Pull request by cskksc**
([A]/[B]/[C]) are spaced out to look like this: ( [A] / [B] / [C] )

Used this [script](https://gist.github.com/cskksc/cd410ab4a3e45351146e).

Fixes #1407

### CODE REVIEW

**Comment by cskksc** [comment]
Work in progress. Just started with the json files.

**Comment by moollaza** [comment]
@cskksc thanks a lot for this! Feel free to ping me whenever it's ready for review.

**Comment by cskksc** [comment]
@moollaza I think all files in the cheatsheets folder are covered. Please check if I missed any.

**Comment by cskksc** [comment]
@moollaza ping

**Comment by moollaza** [comment]
> @moollaza ping

Thanks! Checking it out right now

File: share/goodie/cheat_sheets/json/mac-keyboard.json [hunk 1]
```diff
@@ -115,7 +115,7 @@
             "key" : "[Cmd] [,]",
             "val" : "Open Preferences (if available)"
         }, {
-            "key" : "[Ctrl] [←/→]",
+            "key" : "[Ctrl] [ ← / → ]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/mac-keyboard.json]: These arrows should be wrapped in `[]` with a `()` around the pair. I can fix...

**Comment by moollaza** [comment]
@cskksc LGTM :+1: 

Thanks a ton!

**Comment by cskksc** [comment]
No problem. Thanks :smiley: .

**Pull request merged**